.pep8.rc

```diff
@@ -0,0 +1,6 @@
+[pep8]
+max-line-length = 99
+show-source = enable
+statistics = enable
+count = enable
+exclude = firefox_pupprteer/docs
```

Comment:
There is still the typing error included. Please fix that and also add an empty final line to that file.